boxes/__init__.py

```diff
@@ -24,6 +24,8 @@
 from contextlib import contextmanager
 import copy
 from shlex import quote
+from shapely.geometry import *
```

Comment:
New dependencies need to be added to `requirements.txt` and `setup.py`

Reply:
--> done

please double check if this is OK, as I can't check it...

Reply:
Adding it to the pip command in `.travis.yml` should also make the CI pass.
Guess it should also be added as ` python3-shapely` to the dnf command in `scripts/Dockerfile`

May be I should put that list somewhere in the docs...